manifests/init.pp

```diff
@@ -15,11 +15,16 @@
   $user                 = $::caddy::params::user,
   $group                = $::caddy::params::group,
   $install_method       = $::caddy::params::install_method,
-  $release_file_name    = $::caddy::params::release_file_name,
```

Comment:
make sure this parameter is not used in any of our current setups. Also for `bin_file_name`